libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/FlyoutObjectDetection.tsx

```diff
@@ -76,8 +76,8 @@ export class FlyoutObjectDetection extends React.Component<
       return <div />;
     }
     const classNames = flyoutStyles();
-    const predictedY = getJoinedLabelString(item?.predictedY);
-    const trueY = getJoinedLabelString(item?.trueY);
+    const correctDetections = getJoinedLabelString(item?.odCorrect);
+    const incorrectDetections = getJoinedLabelString(item?.odIncorrect);
 
     return (
       <FluentUI.FocusZone>
```

Comment:
I see some warnings like:
![image](https://github.com/microsoft/responsible-ai-toolbox/assets/24683184/ede78931-4d82-4134-94e7-32769fa5a2a5)
any idea what is causing these?

Reply:
seems like builds are failing on these as well